### PR TITLE
RMMIS-5738: Added dependency support to Content component

### PR DESCRIPTION
**Pull request by kjschulz**

### CODE REVIEW

**Comment by traviscooperscdhhs** [comment]
:+1:

File: src/Content.jsx [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+import DependencyMixin from './DependencyMixin';
 
 module.exports = React.createClass({
 
   displayName: 'Content',
 
+  mixins: [DependencyMixin],
+
   propTypes: {
     'content' : React.PropTypes.string
```
Comment (mallenscdhhs) [inline src/Content.jsx]: please do not surround object keys with String quotes unless using a reserved keyword or hyphens.

File: src/Content.jsx [hunk 2]
```diff
@@ -25,9 +29,16 @@ module.exports = React.createClass({
     return null;
   },
 
+  getClassNames: function(){
+    return setClassNames(
+      'page-content',
+      {'hidden': !this.state.visible}
```
Comment (mallenscdhhs) [inline src/Content.jsx]: Please remove String quotes from Object key `hidden`.

File: src/Content.jsx [hunk 3]
```diff
@@ -25,9 +29,16 @@ module.exports = React.createClass({
     return null;
   },
 
+  getClassNames: function(){
```
Comment (mallenscdhhs) [inline src/Content.jsx]: feel free to use ES6 Object literal syntax here:
```javascript
getClassNames() {
...
}
```

File: src/Content.jsx [hunk 4]
```diff
@@ -25,9 +29,16 @@ module.exports = React.createClass({
...
+    return setClassNames(
+      'page-content',
+      {'hidden': !this.state.visible}
+    );
+  },
+
   render: function(){
```
Comment (mallenscdhhs) [inline src/Content.jsx]: Please use ES6 Object literal syntax here: 
```javscript
render() {
...
}
```

File: src/Content.jsx [hunk 5]
```diff
@@ -1,10 +1,14 @@
 'use-strict';
-var React = require('react');
+import React from 'react';
+import setClassNames from 'classnames';
+import DependencyMixin from './DependencyMixin';
 
 module.exports = React.createClass({
```
Comment (mallenscdhhs) [inline src/Content.jsx]: Please use ES6 module syntax:
```javascript
export default React.createClass({
...
});
```

File: src/Content.jsx (hunk 1 above)
Comment (kjschulz) [inline src/Content.jsx]: Updated.

File: src/Content.jsx (hunk 2 above)
Comment (kjschulz) [inline src/Content.jsx]: Updated.

File: src/Content.jsx (hunk 3 above)
Comment (kjschulz) [inline src/Content.jsx]: Updated.

File: src/Content.jsx (hunk 4 above)
Comment (kjschulz) [inline src/Content.jsx]: Updated.

File: src/Content.jsx (hunk 5 above)
Comment (kjschulz) [inline src/Content.jsx]: Oh cool, didn't realize I could while using a mixin...updated.

File: src/Content.jsx [hunk 6]
```diff
@@ -15,19 +19,26 @@ module.exports = React.createClass({
     }
   },
 
-  getDefaultProps: function(){
+  getDefaultProps() {
     return {
       'componentType': 'content'
```
Comment (mallenscdhhs) [inline src/Content.jsx]: pls remove the String quotes from `componentType` here.

Comment (kjschulz) [inline src/Content.jsx]: Updated.

**Pull request merged**